modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -20,10 +20,7 @@
 ContentDashboardAdminConfigurationDisplayContext contentDashboardAdminConfigurationDisplayContext = (ContentDashboardAdminConfigurationDisplayContext)request.getAttribute(ContentDashboardWebKeys.CONTENT_DASHBOARD_ADMIN_CONFIGURATION_DISPLAY_CONTEXT);
 %>
 
-<liferay-portlet:actionURL portletConfiguration="<%= true %>" var="configurationActionURL">
-	<portlet:param name="serviceName" value="<%= ContentDashboardAdminConfiguration.class.getName() %>" />
-	<portlet:param name="settingsScope" value="company" />
-</liferay-portlet:actionURL>
+<liferay-portlet:actionURL portletConfiguration="<%= true %>" var="configurationActionURL" />
```

Comment:
Could we move this URL to a Java class? we should try to avoid write logic inside JSP files

Reply:
I think so, I can move this action into the DisplayContext. Thx!

Reply:
please, see https://github.com/liferay-tango/liferay-portal/pull/401/commits/33cb5c0a3981cedf057b2e4518eaa28499d30aa4